wbst-thry/src/app/graphics/lnr-graphics/playground.ts

```diff
@@ -22,23 +36,36 @@ export class Playground {
         skyBoxMaterial.reflectionTexture = new Babylon.CubeTexture('../../assets/blue/blue', scene);
         skyBoxMaterial.reflectionTexture.coordinatesMode = Babylon.Texture.SKYBOX_MODE;
         skyBox.renderingGroupId = 0;
+    }
 
-        // set up central planet
+    private static buildCentralPlanet(scene: Babylon.Scene, highlightLayer: Babylon.HighlightLayer): void {
         let planet = Babylon.MeshBuilder.CreateSphere('planet', { segments: 16, diameter: 4 }, scene);
         planet.renderingGroupId = 1;
+        let planetMaterial = new Babylon.StandardMaterial('planetMaterial', scene);
+        planetMaterial.diffuseTexture = new Babylon.Texture('../../assets/textures/planet1.jpg', scene);
+        planet.material = planetMaterial;
+        planet.actionManager = new Babylon.ActionManager(scene);
+        
+        MeshTriggers.highlightOnHover(planet, highlightLayer, Babylon.Color3.Purple());
+    }
+
+    
 
+    private static buildMoon(scene: Babylon.Scene, highlightLayer: Babylon.HighlightLayer): void {
         // orbit parameters
         let radius = 30;
         let orbitTilt = -.2;
         let speed = .001;
 
-        // set up moon
         let moon = Babylon.MeshBuilder.CreateSphere('moon', { segments: 16, diameter: 1}, scene);
-        moon.position.z = -radius;
         moon.renderingGroupId = 1;
-
+        let moonMaterial = new Babylon.StandardMaterial('moonMaterial', scene);
+        moonMaterial.diffuseTexture = new Babylon.Texture('../../assets/textures/moon.jpg', scene);
```

Comment:
Need to figure out how we can orient the nicer side of the moon towards us. Also is there a way to implement spin? Might be a nice addition